test/parallel/test-timers-same-timeout-wrong-list-deleted.js

```diff
@@ -56,25 +56,27 @@ const handle1 = setTimeout(common.mustCall(function() {
     }));
   }), 10);
 
-  // Make sure our timers got added to the list.
-  const activeHandles = process._getActiveHandles();
-  const activeTimers = activeHandles.filter(function(handle) {
-    return handle instanceof Timer;
-  });
-  const shortTimer = activeTimers.find(function(handle) {
-    return handle._list.msecs === 10;
-  });
-  const longTimers = activeTimers.filter(function(handle) {
-    return handle._list.msecs === TIMEOUT;
-  });
+  setImmediate(() => {
+    // Make sure our timers got added to the list.
+    const activeHandles = process._getActiveHandles();
+    const activeTimers = activeHandles.filter(function(handle) {
```

Comment:
extremely minor nit: mixing arrow functions and regular functions in the added code is a bit inconsistent.